packages/lighthouse-plugin-amp/README.md

```diff
@@ -0,0 +1,11 @@
+# Lighthouse AMP plugin
+
+[![npm version](https://badge.fury.io/js/%40ampproject%2Ftoolbox-cors.svg)](https://badge.fury.io/js/%40ampproject%2Ftoolbox-cors)
```

Comment:
Right, removed link as for the current packages, it links to the npm package page. This will not exist for lighthouse plugin. I am following suit with other plugins. Speaking with lighthouse group about their current work on how non default plugins will surface for users.